lib/bolt/plugin.rb

```diff
@@ -36,6 +36,12 @@ def initialize(plugin_name, hook)
           super("Plugin #{plugin_name} does not support #{hook}", 'bolt/unsupported-hook')
         end
       end
+
+      class PluginLoadingDisabled < PluginError
+        def initialize(plugin_name)
+          super("Cannot load plugin #{plugin_name}: plugin loading is disabled", 'bolt/plugin-loading-disabled')
```

Comment:
Can we put the plugin name in structured data (details hash maybe) so that we can use it easily when we convert it when we catch this for `PluginNotSupported` errors?

Reply:
Done.